distributions/openhab/src/main/resources/conf/services/addons.cfg

```diff
@@ -3,18 +3,13 @@
 # Note: If you want to specify your add-ons yourself through entries below, set the package to "minimal"
 # as otherwise your definition might be in conflict with what the installation package defines.
 #
-# Optional. If not set, the dashboard (https://<yourserver>:8080/) will ask you to choose a package.
-#
 # Valid options:
 #   - minimal  : Installation only with dashboard, but no UIs or other add-ons. Use this for custom setups.
```

Comment:
Leftover reference to the dashboard here.
```suggestion
#   - minimal  : Installation only with the UI but no other add-ons. Use this for custom setups.
```

Reply:
Not even with the UI - the idea for the minimal setup was to use it e.g. for a headless system. Will fix the formulation.